src/pudl/metadata/resources/ferc714.py

```diff
@@ -97,6 +97,27 @@
         "field_namespace": "ferc714",
         "etl_group": "outputs",
     },
+    "core_ferc714__yearly_planning_area_demand_forecast": {
+        "description": (
+            "10-year forecasted summer and winter peak demand and annual net energy per planning area. FERC Form 714, Part III, "
+            "Schedule 2b."
+        ),
+        "schema": {
+            "fields": [
+                "respondent_id_ferc714",
+                "report_year",
+                "forecast_year",
+                "summer_peak_demand_mw",
+                "winter_peak_demand_mw",
+                "net_demand_mwh",
+            ],
+            "primary_key": ["respondent_id_ferc714", "report_year", "forecast_year"],
+        },
+        "sources": ["ferc714"],
+        "field_namespace": "ferc714",
+        "etl_group": "ferc714",
+        "create_database_schema": False,
```

Comment:
I think we **do** want to create the database schema here don't we? Since we're writing to the DB as well as Parquet.

Reply:
Updating now

Reply:
@zaneselvans Running into an issue. I updated the `create_database_schema` value above to `True` and ran `alembic revision --autogenerate -m "Updated yearly planning schema" && alembic upgrade head`, which created a migration. 

But now when I try to materialize the new `core_ferc714__yearly_planning_area_demand_forecast` asset, I'm getting this error:

```
ValueError: core_ferc714__yearly_planning_area_demand_forecast 20/17920 duplicate primary keys (pk=['respondent_id_ferc714', 'report_year', 'forecast_year']) when enforcing schema.
```

Do you have any ideas about what I'm doing wrong here?

Reply:
@zaneselvans Running into an issue. I updated the `create_database_schema` value above to `True` and ran `alembic revision --autogenerate -m "Updated yearly planning schema" && alembic upgrade head`, which created a migration. 

But now when I try to materialize the new `core_ferc714__yearly_planning_area_demand_forecast` asset, I'm getting this error:

```
ValueError: core_ferc714__yearly_planning_area_demand_forecast 20/17920 duplicate primary keys (pk=['respondent_id_ferc714', 'report_year', 'forecast_year']) when enforcing schema.
```

Do you have any ideas about what I'm doing wrong here?